transformations/transformations.js

```diff
@@ -1,16 +1,12 @@
 import { HOST_URL } from '../yasp.config';
 import moment from 'moment';
 
-    //TODO move to utility module
-function isRadiant(p)
-{
-  return p.player_slot < 128;
-}
+// TODO move to utility module
+const isRadiant = (p) => p.player_slot < 128;
 
-function pad(n, width, z) {
-    z = z || '0';
-    n = n + '';
-    return n.length >= width ? n : new Array(width - n.length + 1).join(z) + n;
+function pad(n, width, z = '0') {
```

Comment:
This was ported from existing code.  Be absolutely sure it works if you want to refactor